src/hera.cpp

```diff
@@ -268,21 +269,21 @@ void execute(
   instance.callExport(main, args);
 }
 
-void evm_destroy_result(evm_result const* result)
+void hera_destroy_result(evmc_result const* result)
 {
   delete[] result->output_data;
 }
 
-evm_result evm_execute(
-  evm_instance* instance,
-  evm_context* context,
-  enum evm_revision rev,
-  const evm_message* msg,
-  const uint8_t* code,
-  size_t code_size)
-{
-  evm_result ret;
-  memset(&ret, 0, sizeof(evm_result));
+evmc_result hera_execute(
+  evmc_instance *instance,
+  evmc_context *context,
+  enum evmc_revision rev,
+  const evmc_message *msg,
+  const uint8_t *code,
+  size_t code_size
+) {
+  evmc_result ret;
+  memset(&ret, 0, sizeof(evmc_result));
```

Comment:
Is this actually required in C++ or will `ret` be zero initialised anyway?

Reply:
It's not 0 initialized (because it does not have a constructor, it's a C struct).
But in C++ you can do `evmc_result ret = {};` or `evmc_result ret{}`.
In C you can do `evmc_result ret = {EVM_FAILURE}`.